relay/chunkstore/chunk_writer.go

```diff
@@ -104,11 +93,11 @@ func (c *chunkWriter) ProofExists(ctx context.Context, blobKey corev2.BlobKey) b
 	return false
 }
 
-func (c *chunkWriter) CoefficientsExists(ctx context.Context, blobKey corev2.BlobKey) (bool, *encoding.FragmentInfo) {
+func (c *chunkWriter) CoefficientsExists(ctx context.Context, blobKey corev2.BlobKey) bool {
```

Comment:
Are you actually accomplishing anything with this method anymore?

It used to list objects, and then do some kind of checking. Now, it just lists whatever is in the bucket, and returns `true` if there was no error. It's not clear to me that this method would ever return `false`